api-documentation.md

```diff
@@ -166,3 +166,8 @@ Można to skopiować do edytora tekstowego i zapisać jako .csv zachowując odpo
 
 - adres `/hr` metoda: GET
 - adres `/hr/:id` metoda: GET
+
+## 12. Pobieranie siebie
+
+- adres `/user` metoda GET
+- pobiera swoją encję za pomocą id usera z requestu
```

Comment:
w takim info zawsze przyda się odnośnik do typu / interfejsu  (jeśli istnieje) lub po prostu opis zwracanego obiektu json

Reply:
ale zwraca zawsze coś innego, jak jesteś adminen to tylko usera, jak studentem to usera, student initail i profile a jak hr to usera i hr